quarkus-app/src/main/resources/templates/EventResource/cfp.html

```
@@ -308,11 +340,14 @@ <h2 style="margin-top: 0;">{i18n:events_cfp_submit_title}</h2>
     <div style="display: flex; justify-content: space-between; align-items: center; gap: 0.8rem; flex-wrap: wrap;">
       <div>
         <h2 style="margin: 0;">{i18n:events_cfp_my_submissions_title}</h2>
-        {#if userAuthenticated}
+        {#if cfpPreviewMode}
+        <p id="cfpCountLabel" class="form-hint cfp-count-label">{#if resolvedLocaleCode == 'es'}Vista previa{#else}Preview{/if}</p>
+        {/if}
+        {#if !cfpPreviewMode && userAuthenticated}
         <p id="cfpCountLabel" class="form-hint cfp-count-label">0/2 {i18n:events_cfp_proposals_unit}</p>
```

HTMLHint (1.9.2)

[error] The id value [ cfpCountLabel ] must be unique.

(id-unique)